packages/sitecore-jss-react/src/components/PlaceholderCommon.tsx

```diff
@@ -102,6 +109,7 @@ export class PlaceholderCommon<T extends PlaceholderProps> extends React.Compone
       PropTypes.func as Requireable<React.FC<unknown>>,
     ]),
     modifyComponentProps: PropTypes.func,
+    sitecoreContext: PropTypes.object as Requireable<SitecoreContextValue>,
```

Comment:
is there a way to use _PropTypes.object.isRequired_? this is the expected property

Reply:
I updated it but then it breaks the Placeholder component. I think we need to ensure that the PropTypes in PlaceholderCommon component perfectly mirror the TypeScript interfaces.